File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:

        palind=[]
        if(x<0):
            return False
        while(x!=0):
            palind.append(x%10)
            x=int(x/10)

        if(palind==palind[::-1]):
            return True
        else:
            return False
        

File: 0009-palindrome-number/0009-palindrome-number.c
bool isPalindrome(int x) {
        int temp;
        int a=x;
        long long b=0;
        if(x<0){
            return false;
        }
        while(a){
           temp=a%10;
           b=b*10+temp;
           a=a/10; 
        }
        if(x==b){
            return true;
        }else{
            return false;
        }
}